### PR TITLE
Add possibility to specify custom ryuk image

### PR DESCRIPTION
**Pull request by sirdir**
In testcontainers-java there is a possibility to customizing Ryuk resource reaper
https://www.testcontainers.org/features/configuration/#customizing-ryuk-resource-reaper
mainly to use private registry for containers pooling(as in my case)

### CODE REVIEW

File: src/reaper.ts [hunk 1]
```diff
@@ -61,7 +75,10 @@ export class ReaperInstance {
     const sessionId = dockerClient.getSessionId();
 
     log.debug(`Creating new Reaper for session: ${sessionId}`);
-    const container = await new GenericContainer(this.IMAGE_NAME, this.IMAGE_VERSION)
+    const container = await new GenericContainer(
+      ReaperInstance.resolveRyukImageName(),
```
Comment (cristianrgreco) [inline src/reaper.ts:79]: could we change `ReaperInstance` -> `this`?

File: src/reaper.ts [hunk 2]
```diff
@@ -50,6 +50,20 @@ export class ReaperInstance {
...
   }
 
+  private static resolveRyukImageName(): string {
+    if (process.env.RYUK_CONTAINER_IMAGE !== undefined) {
+      this.IMAGE_NAME = process.env.RYUK_CONTAINER_IMAGE.split(/:/)[0];
+    }
+    return this.IMAGE_NAME;
+  }
+
+  private static resolveRyukImageVersion(): string {
```
Comment (cristianrgreco) [inline src/reaper.ts:60]: could we rename to `getImageName()` and `getImageVersion()`?

File: README.md [hunk 3]
```diff
@@ -23,6 +23,7 @@ The following environment variables are supported:
 | `DEBUG` | `testcontainers*` | See all output |
 | `DOCKER_HOST` | `tcp://docker:2375` | Override the Docker host |
 | `TESTCONTAINERS_RYUK_DISABLED` | `true` | Disable [ryuk](#ryuk) |
+| `RYUK_CONTAINER_IMAGE` | `registry.mycompany.com/mirror/ryuk:0.3.0` | Custom image name for [ryuk](#ryuk) |
```
Comment (cristianrgreco) [inline README.md:26]: could we change "Custom image ~~name~~"

**Comment by cristianrgreco** [comment]
thanks for raising @sirdir ! just a couple of nitpicks

File: README.md (hunk 3 above)
Comment (sirdir) [reply README.md]: done

**Review by cristianrgreco — APPROVED** [review]

**Comment by cristianrgreco** [comment]
released in https://github.com/testcontainers/testcontainers-node/releases/tag/v6.3.0, thanks @sirdir !

**Pull request merged**